clang-tidy/action.yaml

```diff
@@ -115,28 +109,43 @@ runs:
         if [ "${{ inputs.target-files }}" != "" ]; then
           target_files="${{ inputs.target-files }}"
         else
-          package_path=$(colcon list --paths-only --packages-select ${{ inputs.target-packages }})
-          target_files=$(find $package_path -type f \( -name '*.cpp' -o -name '*.hpp' \) -not -path '*/test/*' -not -path '*/examples/*')
+          mapfile -t package_paths < <(colcon list --paths-only --packages-select ${{ inputs.target-packages }})
+          ignore_patterns=()
+          ignore_patterns+=( -not -path "*/test/*" )
+          if [ -f .clang-tidy-ignore ]; then
```

Comment:
I think it's better if we make the file path an input argument for this action so that users can place the ignore file anywhere thy want.

Reply:
Thank you! I agree. I will change it.

Reply:
Changed!